src/controllers/User.js

```diff
@@ -0,0 +1,126 @@
+const UserModel = require('../models/user');
+const SignUpValidator = require('../middlewares/validators/SignUpValidator');
+const user = require('../models/user');
+const jwt = require("jsonwebtoken");
+const nodemailer = require("nodemailer");
+const jwtDecode = require('jwt-decode');
+
+class User {
+    static signUp = async (request, response) => {
+        //Validation of data
+        // (async () => {
```

Comment:
Don't leave commented code in your PR. If you don't need this function then remove it.